index.html

```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
           <a href="#">Menu <span class="fa fa-bars"></span></a>
         </div>
 
+
+<!--BODY, BUILT BY CMOORE-->
+<body>
```

Comment:
The opening `<body>` tag needs to be below the closing `</head>` tag and above the `<!--HEADER, BUILT BY CAMPER-->` comment on line 14. If we merge this code the way it is now, with the opening body tag where it is, it'll overwrite the correct code currently in the staging branch, essentially creating a bug.